rotkehlchen/db/dbhandler.py

```diff
@@ -2358,6 +2358,41 @@ def delete_bitcoin_xpub(self, xpub_data: XpubData) -> None:
         self.conn.commit()
         self.update_last_write()
 
+    def edit_bitcoin_xpub(self, xpub_data: XpubData) -> None:
+        """Edit the xpub tags and label
+
+        May raise:
+        - InputError if the xpub data already exist
+        """
+        cursor = self.conn.cursor()
+        key = xpub_data.xpub.xpub + xpub_data.serialize_derivation_path_for_db()  # type: ignore
+        try:
+            # Delete the tag mappings for the xpub itself (type ignore is for xpub is not None
```

Comment:
parentheses is not closing. I know its copy pasta error from the function above. But also the type ignore comment refers to the line above since you moved things.